config/settings.yml

```
@@ -3,3 +3,6 @@ user:
   email_max_length: 255
   password_min_length: 6
   VALID_EMAIL_REGEX: <%= /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i %>
+course:
+  name_max_length: 255
+  description_max_length: 1000
```

```suggestion
length:
  name_max_255: 255
  max_1000: 1000
```

đặt biên như ri để còn tái sử dụng dc bé bì nhé